src/react-apps/applications/runtime/src/sagas/formFiller/formFillerSagas.ts

```diff
@@ -185,11 +185,10 @@ export function* uploadAttachmentSaga(
     const servicePath = `${org}/${service}`;
     const data = new FormData();
     data.append('file', file);
-    const url = `${altinnWindow.location.origin}/runtime/api/${reportee}/` +
-      `${servicePath}/GetAttachmentUploadUrl/${instanceId}/${attachmentType}/${file.name}`;
 
-    const fileUploadLink = yield call(get, url);
-    const response = yield call(post, fileUploadLink, null, data);
+    // tslint:disable-next-line:max-line-length
+    const fileUploadUrl = `${altinnWindow.location.origin}/runtime/api/attachment/${reportee}/${servicePath}/${instanceId}/SaveFormAttachment?attachmentType=${attachmentType}&attachmentName=${file.name}`;
```

Comment:
The common path to all the actions (`${altinnWindow.location.origin}/runtime/api/attachment/${reportee}/${servicePath}/${instanceId}) could be saved in a variable and reused below

Reply:
This is the runtime code that Jesper is rewriting, no point in using time to correct it since this code is not going to be used once the new runtime is in use.